src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -7637,6 +7637,9 @@ static void M1(B b)
             // https://github.com/dotnet/roslyn/issues/44691
```

Comment:
>// https://github.com/dotnet/roslyn/issues/44691 [](start = 12, length = 48)

This issue was closed. Consider adding a variant of this test that also has an attribute on the property.

Reply:
Done